src/docs/Foundations/Layout/Grid/code.mdx

```diff
@@ -24,43 +19,24 @@ order: 1
 @import 'layouts/_l.flexy';
 ```
 
-`ml-flexy` is very close to the bootstrap flex grid system.
+### How it works
 
-To create a grid, you need two elements:
+The below example creates four equal-width columns at small, medium, large, and extra large breakpoints using our predefined grid classes. The columns are centered in the page within the parent container.
```

Comment:
~~The columns are centered in the page within the parent container.~~ not necessary